packages/fuel-gauge/src/transaction-summary.test.ts

```diff
@@ -591,5 +591,45 @@ describe('TransactionSummary', () => {
         recipients: allRecipients,
       });
     });
+
+    it('should ensure that transfer operations are assembled correctly if only seeded with a MessageInput (SPENDABLE MESSAGE)', async () => {
+      const testMessage = new TestMessage({ amount: 1000000, data: '' });
```

Comment:
@maschad Do you think it makes sense also to validate if this operation is being returned as expected when funded with UTXOs only? When investigating this issue at my side, using master, I could see that this operation was not working ( at least not always ) when the TX was funded with UTXOs.

Reply:
Maybe I am misinterpreting your suggestion @Torres-ssf but isn't that what the preceding [two tests](https://github.com/FuelLabs/fuels-ts/blob/mc/fix/ensure-correct-input-message-returned/packages/fuel-gauge/src/transaction-summary.test.ts#L448-L593) are testing for?